src/main/java/com/ita/if103java/ims/dao/impl/UserDaoImpl.java

```diff
@@ -204,9 +204,12 @@ private PreparedStatement getPreparedStatement(User user, Connection connection)
 
         static final String SQL_SELECT_ALL_USERS = "SELECT * FROM users";
 
-        static final String SQL_SELECT_USER_BY_ACCOUNT_ID = "SELECT * FROM users WHERE account_id = ?";
+        static final String SQL_SELECT_USERS_BY_ACCOUNT_ID = ""+
+            "SELECT * FROM users WHERE id IN " +
+            "(SELECT admin_id FROM accounts WHERE  account_id = ?)";
```

Comment:
Why do we need that subselect here?
```mysql
 (SELECT admin_id FROM accounts WHERE  account_id = ?)
```
Can it be just:
```mysql
SELECT * FROM users WHERE account_id = ?
```

Reply:
It is for another method when we want to find an account admin but for now, I've not done this method yet.

Reply:
This query should work, isn't it?
`SELECT * FROM users WHERE role = 'ADMIN' AND account_id = ?`

Reply:
You are right